packages/realm/rollup.config.mjs

```diff
@@ -53,7 +57,7 @@ export default [
         tsconfig: "src/node/tsconfig.json",
       }),
     ],
-    external: ["bson", "debug", "node:module", "node:fs", "node:path", "node:http", "node:https"],
+    external: ["bson", "debug", "node-fetch", "node:module", "node:fs", "node:path"],
```

Comment:
We're bundling in the `@realm/network-transport`, so it won't be a dependency of the package, with the risk of mismatching versions.